### PR TITLE
Fixes NPE when importing string

### LINKED ISSUE #2013: Syntax linting of import causes NullPointerException
v2023.02.17

native

CLI

For the invalid Clojure snippet `(import "")` I get an NPE. I would have expected a more descriptive error message.

```
❯ echo '(import "")' | clj-kondo --lint -
<stdin>:0:0: error: Can't parse <stdin>, java.lang.NullPointerException
linting took 39ms, errors: 1, warnings: 0
```

```
(import "")
```
output:
```
<stdin>:0:0: error: Can't parse <stdin>, java.lang.NullPointerException
```

I would have expected an error message saying this is not valid Clojure.

I would want to try and fix this myself.

### PR DESCRIPTION
**Pull request by cosineblast**
Please answer the following questions and leave the below in as part of your PR.

- [X] I have read the [developer documentation](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md).

- [X] This PR corresponds to an [issue with a clear problem statement](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [X] This PR contains a [test](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#tests) to prevent against future regressions

- [X] I have updated the [CHANGELOG.md](https://github.com/clj-kondo/clj-kondo/blob/master/CHANGELOG.md) file with a description of the addressed issue.

This PR fixes #2013, linting `(import "")` or similar constructs no longer causes a NullPointerException.

### CODE REVIEW

**Comment by borkdude** [comment]
@cosineblast Thanks, can you update the CHANGELOG.md to start a new `Unreleased` section?

File: test/clj_kondo/main_test.clj [hunk 1]
```diff
@@ -3420,6 +3420,14 @@ foo/")))
   (is (= "error" (:fail-level (#'clj-kondo.main/parse-opts ["--fail-level" "error"]))))
   (is (= "error" (:fail-level (#'clj-kondo.main/parse-opts ["--fail-level=error"])))))
 
+(deftest import-illegal-token
+  (lint! "(import \"\")")
```
Comment (borkdude) [inline test/clj_kondo/main_test.clj:3424]: What is this expression doing?

Comment (cosineblast) [reply test/clj_kondo/main_test.clj:3424]: My bad, I was using it when testing things with CIDER, but left it by accident. I'm pushing a fix for that.

**Comment by borkdude** [comment]
Thanks @cosineblast !

**Pull request merged**